srcpkgs/php8.0-apcu/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'php8.0-apcu'
+pkgname=php8.0-apcu
+version=5.1.21
+revision=1
+wrksrc="apcu-${version}"
+build_style=gnu-configure
+configure_args="--with-php-config=/usr/bin/php-config8.0"
+make_check_target=test
+hostmakedepends="autoconf php8.0-devel"
+makedepends="php8.0-devel pcre2-devel"
+depends="php8.0>=8.0.16"
```

Comment:
Why is this version restriction necessary?

Reply:
Dropped version restrictions.